### PR TITLE
Add ekubo pools

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

**Comment by Chepelau** [comment]
This PR adds Arbitrary Proposal for adding ekubo pools (**not options too!!**).  Please check it, as well as the fork test.

In `constants.cairo`, there is an updated amm class hash. Please check that the hash is the same as when building branch `add-ekubo-options` in our amm repo. 

@tensojka  @DaveVodrazka

**Comment by Chepelau** [comment]
Also commented out `test_add_custom_proposal` fork test because it's outdated and failing on `already voted`.

**Review by DaveVodrazka — APPROVED** [review]
Got the same class-hash for the AMM 👍

File: tests/add_ekubo_pools.cairo [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    loop {
+        match voter_addresses.pop_front() {
+            Option::Some(address) => {
+                let current_voter: ContractAddress = (*address).try_into().unwrap();
+                prank(CheatTarget::One(gov_addr), current_voter, CheatSpan::TargetCalls(1));
+                props.vote(prop_id, 1);
+            },
+            Option::None(()) => { break (); }
+        }
+    };
```
Comment (ondrejsojka) [inline tests/add_ekubo_pools.cairo:82]: did you know the famed for loop has come to our land?

https://community.starknet.io/t/cairo-v2-7-0-is-coming/114362

(not worth changing, just wanted to mention)

File: src/arbitrary_proposal_add_ekubo_pools.cairo [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            let (call_lpt_address, _) = call_deploy_retval.unwrap_syscall();
+            // Put deploy
+            let put_deploy_retval = deploy_syscall(
+                lptoken_class, 'ekubousdc put', ekubo_put_lpt_calldata.span(), false
+            );
+            let (put_lpt_address, _) = put_deploy_retval.unwrap_syscall();
+
+            // Add the lptokens to the AMM
+            let call_voladjspd = FixedTrait::new_unscaled(10_000, false);
+            let put_voladjspd = FixedTrait::new_unscaled(50_000, false);
```
Comment (ondrejsojka) [inline src/arbitrary_proposal_add_ekubo_pools.cairo]: I think this is too high, DMd

**Review by ondrejsojka — APPROVED** [review]
I would adjust the voladjspd, but otherwise lgtm!

**Review by ondrejsojka — APPROVED** [review]

**Pull request merged**